conf/messages

```diff
@@ -1176,139 +1176,256 @@ acquisitionDetails.caption=Purchase or acquisition details
 acquisitionPriceBought.trust.error.tooSmall=How much the trust paid for the property must be an amount of money between 1 and 50,000,000,000
-acquisitionPriceNotBought.error.tooSmall=The market value must be an amount of money between 1 and 50,000,000,000
-acquisitionPriceNotBought.agent.error.tooSmall=The market value must be an amount of money between 1 and 50,000,000,000
-acquisitionPriceNotBought.trust.error.tooSmall=The market value must be an amount of money between 1 and 50,000,000,000
+acquisitionPriceBought.capacitor.error.tooSmall=How much the person paid for the property must be an amount of money between 1 and 50,000,000,000
+acquisitionPriceBought.personalRep.error.tooSmall=The market value of the property must be an amount of money between 1 and 50,000,000,000
+
 acquisitionPriceBought.error.tooLarge=How much you paid for the property must be an amount of money between 1 and 50,000,000,000
 acquisitionPriceBought.agent.error.tooLarge=How much your client paid for the property must be an amount of money between 1 and 50,000,000,000
 acquisitionPriceBought.trust.error.tooLarge=How much the trust paid for the property must be an amount of money between 1 and 50,000,000,000
-acquisitionPriceNotBought.error.tooLarge=The market value must be an amount of money between 1 and 50,000,000,000
-acquisitionPriceNotBought.agent.error.tooLarge=The market value must be an amount of money between 1 and 50,000,000,000
-acquisitionPriceNotBought.trust.error.tooLarge=The market value must be an amount of money between 1 and 50,000,000,000
+acquisitionPriceBought.capacitor.error.tooLarge=How much the person paid for the property must be an amount of money between 1 and 50,000,000,000
+acquisitionPriceBought.personalRep.error.tooLarge=The market value of the property must be an amount of money between 1 and 50,000,000,000
+
 acquisitionPriceBought.error.tooManyDecimals=How much you paid for the property must be an amount of money with pence as 2 decimal places or less, like 160,000 or 23,040.50
 acquisitionPriceBought.agent.error.tooManyDecimals=How much your client paid for the property must be an amount of money with pence as 2 decimal places or less, like 160,000 or 23,040.50
 acquisitionPriceBought.trust.error.tooManyDecimals=How much the trust paid for the property must be an amount of money with pence as 2 decimal places or less, like 160,000 or 23,040.50
+acquisitionPriceBought.capacitor.error.tooManyDecimals=How much the person paid for the property must be an amount of money with pence as 2 decimal places or less, like 160,000 or 23,040.50
+acquisitionPriceBought.personalRep.error.tooManyDecimals=The market value of the property must be an amount of money with pence as 2 decimal places or less, like 160,000 or 23,040.50
+
+
+acquisitionPriceNotBought.title=What was the market value of the property on {0}?
```

Comment:
shouldn't this be `What was the market value of the property when it was acquired?` This looks like the title that should go on the rebasedAcquisitionPrice page

Reply:
I think so, but not sure as its existing message. We have to check with Jean